src/pages/workspace/WorkspaceProfilePlanTypePage.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import Button from '@components/Button';
+import FixedFooter from '@components/FixedFooter';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import CardSectionUtils from '@pages/settings/Subscription/CardSection/utils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import AccessOrNotFoundWrapper from './AccessOrNotFoundWrapper';
+import withPolicy from './withPolicy';
+import type {WithPolicyProps} from './withPolicy';
+
+type WorkspacePlanTypeItem = {
+    value: ValueOf<typeof CONST.POLICY.TYPE>;
+    text: string;
+    alternateText: string;
+    keyForList: ValueOf<typeof CONST.POLICY.TYPE>;
+    isSelected: boolean;
+};
+function WorkspaceProfilePlanTypePage({policy}: WithPolicyProps) {
+    const [currentPlan, setCurrentPlan] = useState(policy?.type);
+    const policyID = policy?.id ?? '-1';
+    const {translate} = useLocalize();
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
+
+    const workspacePlanTypes = Object.values(CONST.POLICY.TYPE)
+        .filter((type) => type !== CONST.POLICY.TYPE.PERSONAL)
+        .map<WorkspacePlanTypeItem>((policyType) => ({
+            value: policyType,
+            text: translate(`workspace.planTypePage.planTypes.${policyType as Exclude<typeof policyType, 'personal'>}.label`),
+            alternateText: translate(`workspace.planTypePage.planTypes.${policyType as Exclude<typeof policyType, 'personal'>}.description`),
+            keyForList: policyType,
+            isSelected: policyType === currentPlan,
+        }))
+        .reverse();
+
+    const isControl = policy?.type === CONST.POLICY.TYPE.CORPORATE;
+    const isAnnual = privateSubscription?.type === CONST.SUBSCRIPTION.TYPE.ANNUAL;
+
+    const isPlanTypeLocked = isControl && isAnnual;
```

Comment:
`isPlanTypeLocked` is not the final logic here, as I'm currently unsure how to determine if a specific policy is on a yearly plan. I've asked for clarification https://expensify.slack.com/archives/C02NK2DQWUX/p1731682677445059, but for now, we can finalize the UI and adjust the show logic later.

Reply:
Before merging, I also wanted to let you know that `isPlanTypeLocked` logic is also not yet confirmed

Reply:
@carlosmiceli ⬆️

Reply:
Got it, if we can add it later then that's all good.

Reply:
I think there's a few relevant answers now in the slack thread :)